config/samples/clusters_v1beta1_cadence.yaml

```diff
@@ -51,6 +51,8 @@ spec:
       nodeSize: "CAD-DEV-t3.small-5"
       nodesNumber: 2
       clientEncryption: false
+#      privateLink:
+#        - advertisedHostname: bohdan-cadence-test.com
```

Comment:
I don't think that it is appropriate to use names here

Reply:
Fixed